staxapp/api.py

```diff
@@ -27,7 +27,9 @@ def get(cls, url_frag, params={}, **kwargs):
         url_frag = url_frag.replace(f"/{Config.API_VERSION}", "")
         url = f"{Config.api_base_url()}/{url_frag.lstrip('/')}"
 
-        response = requests.get(url, auth=cls._auth(), params=params, **kwargs)
+        response = requests.get(
+            url, auth=cls._auth(), params=params, headers=Config.user_agent, **kwargs
```

Comment:
Assigning the user agent headers here would prevent a user from adding any other headers in their request.